sql/core/src/test/scala/org/apache/spark/sql/TPCDSBase.scala

```diff
@@ -36,6 +36,12 @@ trait TPCDSBase extends SharedSparkSession with TPCDSSchema {
     "q81", "q82", "q83", "q84", "q85", "q86", "q87", "q88", "q89", "q90",
     "q91", "q92", "q93", "q94", "q95", "q96", "q97", "q98", "q99")
 
+  // Since `tpcdsQueriesV2_7_0` has almost the same queries with these oens below,
+  // we skip them in the TPCDS-related tests.
+  private val excludedTpcdsQueries: Set[String] = Set("q6", "q34", "q64", "q74", "q75", "q78")
+
+  val tpcdsQueries: Seq[String] = tpcdsAllQueries.filterNot(excludedTpcdsQueries.contains)
```

Comment:
another idea:
```
val tpcdsQueries: Seq[String] = 1.to(99).map("q" + _).flatMap { q => 
  if (Seq("q14", "q23", "q24", "q39").contains(q)) Seq(q + "a", q + "b") else Seq(q)
}.filterNot { q =>
  // ...
  Seq("q6", "q34", "q64", "q74", "q75", "q78").contains(q)
}
```

Reply:
I feel the `if` part looks a bit tricky, so I prefer to keep it as it is.